README.md

```diff
@@ -1,35 +1,82 @@
 # Shared Docker Development Setup for Airflow DAGs
 
 
-This repo contains all of the setup needed to run airflow in docker, mount your local dag repo into the containers that need it, and allow you to develop dags hopefully quickly!
+This Git repository contains all of the setup needed to run airflow in docker, mount your local DAG repo into the containers that need it, and allow you to develop DAGs hopefully quickly!
 
 
 ## Expected Setup
 
 This repo expects:
 
-* to be a git submodule living in the root of your dags repo. 
-* dag files are in a directory in the top level.
-* a `local.env` file with `DAG_DIR=some_name` with the name of the directory containing your dag files.
+* To be a Git submodule residing in the root of your DAG's Git repoitory. 
+* The DAG project directory structure:
```

Comment:
@nomadicoder I find this structure to be confusing.  All of the docker files are shown at the top level rather than in the airflow-docker-dev-setup directory.  I think it would be better to take out the whole directory structure out. You could include links to some/all of the repositories instead.

Reply:
Rewritten